cleanlab/count.py

```diff
@@ -96,18 +85,45 @@ class 0, 1, ..., K-1. `pred_probs` should have been computed using 3 (or
       The `confident_joint` can be computed using :py:func:`count.compute_confident_joint <cleanlab.count.compute_confident_joint>`.
       If not provided, it is computed from the given (noisy) `labels` and `pred_probs`.
 
+    estimation_method :
+      Method for estimating the number of label issues in dataset by counting the examples in the off-diagonal of the `confident_joint` ``P(label=i, true_label=j)``.
```

Comment:
@cgnorthcutt Could you elaborate on this:
> for the off_diagonals method, we should still calibrate the rows to correctly sum to the true prior (p(s)) of noisy given labels (without also normalizing to sum to the total number of examples) -- this would make the rows more accurately affect the prior distribution without overcounting errors. 

The main point of this PR is to make the num_label_issues count match find_label_issues when off_diagonal is the method. Wouldn't any change like you are suggesting make them no longer match?

Reply:
i would still benchmark to see if its more accurate and if it is, then it sure another `method` added to this function called `off_diagonal_row_calibrated`

Initial benchmarks suggest this `off_diagonal` method is more accurate (in addition to matching `find_label_issues(filter_by='confident_learning')`) but may slightly undercount.